drush.services.yml

```diff
@@ -0,0 +1,5 @@
+services:
+  database_sanitize.commands:
+    class: \Drupal\database_sanitize\Commands\DatabaseSanitizeCommands
+    tags:
```

Comment:
Lets inject here the `database_sanitize` service:
`arguments: ['@database_sanitize']`

Then add a constructor like on http://cgit.drupalcode.org/devel/tree/src/Commands/DevelCommands.php

Reply:
good idea - I probably won't get time to work on this over the next few days, so if you have time please could you take this forward?